packages/relay/src/lib/eth.ts

```diff
@@ -1643,9 +1643,11 @@ export class EthImpl implements Eth {
 
     const selector = getFunctionSelector(call.data!);
 
+    // The force-to-consensus-by-selector logic is active only when process.env.ETH_CALL_FORCE_TO_CONSENSUS_BY_SELECTOR is set to 'true'.
+    // When eth_call is invoked with a selector listed in specialSelectors, it will be routed through the consensus node, regardless of ETH_CALL_DEFAULT_TO_CONSENSUS_NODE.
```

Comment:
Can we add a note to say that this is a temporary measure and will be removed once mirror node can handle eth_call for `isAssociated`?

Reply:
Ah yeah I just finished creating [a ticket](https://github.com/hashgraph/hedera-json-rpc-relay/issues/2984) just for that. It's a good idea I'll include the ticket in the note.